http://developer.berlios.de/bugs/?func=detailbug&bug_id=19139&group_id=5358

This patch might not work with wxGTK, but it's needed for wxMAC and wxOSX (2.8 and 2.9).
It needs to be fixed upstream, but a better patch is needed to deal with the problem.
See also http://docs.wxwidgets.org/2.9.5/overview_bitmap.html

--- src/plugins/contrib/NassiShneiderman/NassiView.cpp.orig
+++ src/plugins/contrib/NassiShneiderman/NassiView.cpp
@@ -1048,15 +1048,9 @@
 
     if ( dataptr )
     {
-        #if defined(__WXMSW__)
             wxCursor copycursor(dnd_copy_cur_xpm);
             wxCursor movecursor(dnd_move_cur_xpm);
             wxCursor nonecursor(dnd_none_cur_xpm);
-        #else
-            wxIcon copycursor(dnd_copy_cur_xpm);
-            wxIcon movecursor(dnd_move_cur_xpm);
-            wxIcon nonecursor(dnd_none_cur_xpm);
-        #endif
         //wxDragResult result;
 
         wxDropSource dndSource(m_diagramwindow, copycursor, movecursor, nonecursor);
